agent/tcs/handler/handler.go

```diff
@@ -86,10 +87,9 @@ func StartSession(params TelemetrySessionParams, statsEngine stats.Engine) error
 func startTelemetrySession(params TelemetrySessionParams, statsEngine stats.Engine) error {
 	tcsEndpoint, err := params.ECSClient.DiscoverTelemetryEndpoint(params.ContainerInstanceArn)
 	if err != nil {
-		log.Errorf("Unable to discover poll endpoint: ", err)
+		seelog.Errorf("Unable to discover poll endpoint: ", err)
```

Comment:
This either needs a format string (probably `%v`) or should be `Error` instead of `Errorf`.

Reply:
Agreed. Fixing